pkg/controller/job/metrics/metrics.go

```diff
@@ -109,6 +121,11 @@ const (
 
 	Succeeded = "succeeded"
 	Failed    = "failed"
+
+	// Possible values for "event"  label in the terminated_pods_tracking_finalizer
```

Comment:
```suggestion
	// Possible values for "operation"  label in the terminated_pods_tracking_finalizer
```

Reply:
oops, I actually meant to change the label to event. Unless that's not a good name?

Reply:
No, that's fine, so long as it's consistent.